src/components/Login-class.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import {
+  FormControl,
+  FormGroup,
+  Button,
+  FormLabel,
+  Image,
+} from "react-bootstrap";
+import PCTLogo from "../assets/PCT-logo.png";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #eee;
+`;
+const Title = styled.h2`
+  margin: 10;
+`;
+
+constructor(props) {
```

Comment:
this should go in the class component below, but is this component even being used?